examples/user/thermoelasticity.yaml

```diff
@@ -0,0 +1,25 @@
+name: Thermoelastic deformation of NURBS cylinder
+authors:
+  - Roxana Pohlmann
+description: |
+  This example considers the thermal deformation of a hollow 
+  cylinder. First, a heat problem is solved, where the temperature from the 
+  boundary diffuses into the inner cylinder. The solution to this problem 
+  is used as the initial temperature in a linear thermoelastic problem. 
+  As the resulting temperature - a homogeneous field - is known, the temperature
+  equation does not need to be solved. 
+  More information regarding the theory of elasticity and thermoelasticity 
+  can be found in, for example, ["Elasticity - Therory, Application and 
+  Numerics" by M. Sadd][Sadd2014]
+
+  [Sadd2014]: https://doi.org/10.1016/C2012-0-06981-5
+repository: https://github.com/Roxana-P/nutils-thermoelastic.git
+commit: 8dac5338ca03b2d61145f1261144346e1d5d14ea
+script: thermoelastic-example.py
+images:
+  - displacement.png
+  - deformed_cylinder.vtk
```

Comment:
The images listed here are used on the website unmodified. Since browsers can't display VTKs, can you please remove the VTK from the list?